challenges.yml

```
@@ -1699,3 +1699,448 @@ extending_smart_watcher_2:
+
+          export default {
+              name: 'PizzaCheckout',
+              data() {
+                  return {
+                      showDisclaimer: false,
+                  };
+              },
+              methods: {
+                  async addPizzaToCart() {
+                      await buyPizza();
+                  }
+              },
+          };
+          </script>
+
+          <style lang="scss" module>
+          .component {
+              .v-enter-active, .v-leave-active {
+                  transition: opacity 1s;
+              }
+
+              .v-enter, v-leave {
+                  opacity: 0;
+              }
+          }
+          </style>
+          ```
+        d: |
+          ```vue
+          <template>
+              <div :class="$style.component">
+                  <button @click="addPizzaToCart" />
+
+                  <p v-show="showDisclaimer" transition>
+                      <strong>Disclaimer</strong>: Eating too much pizza can be
+                      bad for your health!
+                  </p>
+              </div>
+          </template>
+
+          <script>
+          import { buyPizza } from '@/services/buy-pizza';
+
+          export default {
+              name: 'PizzaCheckout',
+              data() {
+                  return {
+                      showDisclaimer: false,
+                  };
+              },
+              methods: {
+                  async addPizzaToCart() {
+                      await buyPizza();
+                  }
+              },
+          };
+          </script>
+
+          <style lang="scss" module>
+          .component :global {
+              .v-enter-active, .v-leave-active {
+                  transition: opacity 1s;
+              }
+
+              .v-enter, v-leave {
+                  opacity: 0;
+              }
+          }
+          </style>
+          ```
+      correct: b
+
+extending_transition_name:
+  question: |
+    What does the transition `name` property do?
+  explanation: |
+    Having several transitions forces us to use `names` so that we can have well
+    defined classes defining them!
+  features:
+    multiple_choice:
+      choices:
+        a: |
+          It names the transition so that we can track it in the Vue Dev Tools
+          should we need to debug it
+        b: |
+          It changes the name of the transition classes for a given transition
+          so you can have different transitions in your application with different
+          names
+        c: |
+          Name doesn't really do anything. It's just a standard DOM attribute we
+          can use to select elements in the DOM
+        d: |
+          The `name` property in a transition corresponds to the name of the component
+          they are being used in. It's not mandatory, but it's nice to add it.
+      correct: b
+
+extending_transition_name_2:
+  question: |
+    Vue transitions have also a `mode` attribute. The default mode is `in-out`.
```

```suggestion
    Vue transitions also have a `mode` attribute. The default mode is `in-out`.
```

I think the default is not `in-out`. It is "make both transitions happen at the same time".